fuel-base/src/main/kotlin/fuel/HttpLoader.kt

```diff
@@ -22,6 +22,20 @@ public interface HttpLoader {
     public suspend fun head(request: Request): Response
     public suspend fun method(request: Request): Response
 
+    /**
+     * Lazily load the [request]'s data by representing with [Call]. This is the counterpart of the suspend version above
+     *
+     * @param request The request to execute synchronously.
+     * @return The [Response] result.
+     */
+    public fun getCall(request: Request): Call
+    public fun postCall(request: Request): Call
+    public fun putCall(request: Request): Call
+    public fun patchCall(request: Request): Call
+    public fun deleteCall(request: Request): Call
+    public fun headCall(request: Request): Call
+    public fun methodCall(request: Request): Call
+
```

Comment:
Are these shortcuts for `method()`?

Reply:
Well, it supposed to be the custom http methods. It can be used as get, post and others too.

Reply:
Yeah, as long as there is a way to do non-common HTTP methods (via `method` for example), it's all good.